assets/models/system/tiiuae-falcon-40b/spec.yaml

```diff
@@ -0,0 +1,27 @@
+$schema: https://azuremlschemas.azureedge.net/latest/model.schema.json
+name: tiiuae-falcon-40b
+path: ./
+properties:
+  SHA: c47b371b31a68349c233104050ac76680b8485db
```

Comment:
@choudharykartik please help to confirm SHA and update if modified

Reply:
updated